We do not need to check sources for this project.

--- ddt-1.7.2/tox.ini.orig
+++ ddt-1.7.2/tox.ini
@@ -4,12 +4,10 @@
 [testenv]
 deps = 
     pytest
-    flake8
     six>=1.4.0
     PyYAML
 commands = 
     pytest
-    flake8 ddt.py test
 
 [flake8]
 max-line-length = 127
